src/Arriba/Arriba/Diagnostics/Tracing/ArribaEventSource.cs

```diff
@@ -59,17 +64,21 @@ public interface IArribaEvents : IEventSource
     public sealed class ArribaEventSource : EventSource, IArribaEvents
     {
         public static IArribaEvents Log { get; }
```

Comment:
The duplicate statics trigger me. As we were discussing last week I'd like to get rid of the static logging pattern all together and move to instance based loggers

Reply:
Yes, that's why I marked it as a DRAFT - I'm not expecting this to be long lived. Just a platform for some small discussions.